src/assets/styles/mixins/shared-input-styles.scss

```diff
@@ -45,6 +45,15 @@
       padding: $spacer-micro;
       font-size: $font-size-sm;
     }
+    &::placeholder {
+      color: $color-gray-40;
+    }
+    &::-webkit-input-placeholder { /* Edge */
+      color: $color-gray-40;
+    }
+    &:-ms-input-placeholder { /* Internet Explorer 10-11 */
+      color: $color-gray-40;
```

Comment:
The css autoprefixer might take care of the vendor prefixes, but I haven't confirmed.

Reply:
Thank @bobmanary , you're right, I have just tested and updated it.